### ISSUE TITLE
Increase contrast for accessibility

### ISSUE THREAD

**Issue by mrsantons** [stepzen-samples/stepzen-spacex-graphql#31]
**Is your feature request related to a problem? Please describe.**
Accessibility issues with color contrast making test harder to see for people with accessibility issues. 

**Describe the solution you'd like**
Increase color contrast, if you would still like the lighter grey color the lightest should be #525252
For consistency I would suggest using color #141823

**Additional context**
Desktop: 

<img width="1613" alt="Screen Shot 2021-10-23 at 8 01 08 PM" src="https://user-images.githubusercontent.com/52573981/138575923-0558d652-2566-4e71-913c-1b29d0a92a04.png">
Mobile:
<img width="1751" alt="Screen Shot 2021-10-23 at 7 31 33 PM" src="https://user-images.githubusercontent.com/52573981/138575374-172fd8c0-93cc-490a-9148-9fc1ed3624f6.png">
<img width="1209" alt="Screen Shot 2021-10-23 at 7 39 56 PM" src="https://user-images.githubusercontent.com/52573981/138575547-51af19e5-4712-43ff-904a-3ce88bee9764.png">

After Updating the color should pass all checks as below: 
<img width="1165" alt="Screen Shot 2021-10-23 at 7 42 17 PM" src="https://user-images.githubusercontent.com/52573981/138575598-20b0c701-22a9-4477-8400-eb61e356b719.png">

**Testing Changes**
To test your changes follow the readme instructions on getting started. 

- Once StepZen is running right click and chose inspect:
<img width="565" alt="Screen Shot 2021-10-23 at 7 49 35 PM" src="https://user-images.githubusercontent.com/52573981/138575721-c2e5fe44-530e-49c0-9e53-1e00fa305f1d.png">

- This will open your developer tools and you can then navigate to the lighthouse test, click the blue generate report button and do so for both mobile and desktop
<img width="872" alt="Screen Shot 2021-10-23 at 7 51 22 PM" src="https://user-images.githubusercontent.com/52573981/138575751-8f04fba4-ad1a-4044-b958-9e1b2c6e3651.png">

- Attach a screenshot of the accessibility portion with increased score.
